cmd/init.go

```diff
@@ -241,7 +246,8 @@ func initAppsody(stack string, template string, config *initCommandConfig) error
 			Info.log("If you wish to proceed and overwrite files in the current directory, try again with the --overwrite option.")
 			// this leave the tar file in the dir
 			return errors.Errorf("Error extracting project template: %v", errUntar)
-
+		} else if inputTemplateName != "none" {
```

Comment:
Hi Sandy, sorry for the delay in reviews here.

I don't think this is what the issue intended. I think they wish to alter the existing "Successfully initialized Appsody project" message a few lines down. Change that to be something like:
"Successfully initialized Appsody project with the nodejs-express stack and the default template" or
"Successfully initialized Appsody project with the nodejs-express stack and no template"